runtime/metering.go

```diff
@@ -0,0 +1,59 @@
+package runtime
+
+type MetringOperationType uint
```

Comment:
Both "Operation" and "Type" seem redundant, maybe just `ComputationOperation`?

Reply:
yeah, renamed to the way memoryKind is defined